test/rad/core_test.clj

```diff
@@ -7,19 +7,18 @@
                         [\m \e \a \n \t]
                         [\t \o]
                         [\b \e]
-                        [\h \a \c \k \e \d]
-                        ]))
+                        [\h \a \c \k \e \d]]))
 
 (deftest insertion-tests
   (testing "Determining whether input is meant to go into buffer, or key-commands"
     (is (=
          true
-         (alphanumeric? \a))
+         (alphanumeric? \a)))
 
-        (=
-         false
-         (alphanumeric? :down))
-        (=
+    (is    (=
+            false
+            (alphanumeric? :down)))
+    (is (=
          false
          (alphanumeric? "h")))))
```

Comment:
Should maybe be true? Don't know. Now `alphanumeric?` seems to be the same as `char?`